includes/Admin/Settings_Screens/Product_Sync.php

```diff
@@ -318,6 +318,27 @@ public function get_settings() {
 			],
 			[ 'type' => 'sectionend' ],
 
+			[
+				'name' => __( 'Feed File Settings', 'woocommerce' ),
```

Comment:
Should be sentence case. 

Though we might want to review the UI for this - having it as a regular option in the UI may be confusing if we're trying to encourage merchants to turn it off.